modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network.adoc
+
+[id="update-restricted-network-cli_{context}"]
+= Update the restricted network cluster
+
+After you update the contents of the mirror registry, you can update your
+restricted network cluster.
+
+.Prerequisites
+
+* You have access to the mirror registry that you used to store the images that
+you used to install {product-title}.
+* You have the `ImageContentSourcePolicy` information for the content that you
+mirrored.
+* The the OpenShift Command-line Interface (CLI), commonly known as `oc`, is
+installed.
+* The `jq` package is installed.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the `ImageContentSourcePolicy` resource for your cluster:
++
+----
+$ oc edit ImageContentSourcePolicy
```

Comment:
If just review, I think `oc get ImageContentSourcePolicy -o yaml` will be more save. In QE's testing, we suggest to create new ImageContentSourcePolicy, but not edit old ImageContentSourcePolicy, That would be more easier for track and management. 
There is some discussion in https://jira.coreos.com/browse/CORS-1105?focusedCommentId=115606&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-115606.